src/main/java/spoon/reflect/code/CtComment.java

```diff
@@ -59,12 +60,21 @@
 	String LINE_SEPARATOR = "\n";
 
 	/**
-	 * Get the content of the comment
-	 * @return the content of the comment
+	 * Get the content of the comment.
+	 * The returned content is a parsed version of the raw content:
+	 *   - without the delimiters (/*, //)
+	 *   - without the intermediate prefix " * " in case of Javadoc API comments.
+	 *
+	 * @see #getRawContent() for the raw comment.
+	 * @return the parsed content of the comment
 	 */
-	@PropertyGetter(role = COMMENT_CONTENT)
+	@DerivedProperty
 	String getContent();
 
+	/** The exact string that appears in source code, with delimiters */
+	@PropertyGetter(role = COMMENT_CONTENT)
+	String getRawContent();
```

Comment:
Do we really need to get access to origin raw content? I have bad feeling that it will cause performance problems ... spoon is already often calls `CtElement#equals`, which does a deep compare of whole AST and now each `CtCommentImpl#equals` calls `getContent()` twice ... and getContent always
* cleans raw comment
* parses cleaned comment
* makes a model of comment
* prints text of comment from model

Do we really need raw comment? Cannot we simply store comment in an cleaned and parsed form?

Reply:
We might get raw content as derived property from the origin sources - using SourcePosition ... and to keep model nice and fast with cleaned comment. WDYT?

Reply:
> CtCommentImpl#equals calls getContent()

I am aware of this, I'm not against to undo this change. We  would loose a little bit of backward compatibility (this will require more test changes), but we will gain performance. Do you prefer to go this way?